### ISSUE TITLE
Periodogram() should have power normalisation options

### ISSUE THREAD

**Issue by ojhall94** [KeplerGO/lightkurve#299]
#### Problem description
I've received some feedback from Prof. Tim Bedding that units of Power Spectral Density are useful for solar-type oscillators, but not necessarily for other types. Power Spectral Density should be the default, but other types could be possible.

I'll seek out some more community feedback before building a functional example.
#### Example
```python
pg = lightcurve.to_periodogram(normalization="psd")
```

**Comment by ojhall94** [comment]
Hi Daniel!

Thanks for your great comment--- This is a use case I haven't personally experienced so I'm glad you raise such a clear example. I've had Tim fill me in on the details while he was still here, so I'll try and write a pull request for this soon!

**Comment by barentsen** [comment]
@danielhey Thank you for weighing in!!  Do you have a suggestion as to what Lightkurve's ideal interface might be to enable you to avoid the extra conversions?

**Comment by danielhey** [comment]
Hi all, sorry for the delay in my response!
  
I've been thinking about it and I think a great way for accessing amplitudes instead of fluxes would be an `@property`, something like:
```python
@property
def amplitude(self):
    return self.flux -2.5 * np.log10(self.flux)
```

That way flux is still the default (as it should be!), and the rel. amplitude won't be calculated unless it's needed. Likewise in the `from_lightcurve()` method, maybe a keyword arg could specify the chosen scaling for the spectrum, i.e. normalise=['psd', 'amp', 'power']. 

Attached below are the appropriate rescaling factors for the amplitude and power spectrum:
```python

ndata = len(time)
sc = LombScargle(time, sig).power(fr, normalization='psd')

# 1. if you want to plot the amplitude spectrum
fct = np.sqrt(4./ndata)
amplitude = np.sqrt(sc) * fct

# 2. if you want to have the power spectrum
fct = np.sqrt(4./ndata)
power = fct**2. * sc
```

If you're not working on it already Oliver, I can write up a quick PR.

**Comment by ojhall94** [comment]
Hi Daniel,

If you wanted to do this one that would be great! You've got more experience on this topic so it would almost certainly turn out better than if I did it. Having it as a property is a neat idea, and a kwarg in from_lightcurve() was what I was thinking as well.

Cheers!

**Comment by barentsen** [comment]
I struggle to think of anything more exciting than @danielhey preparing a PR and @ojhall94 testing and reviewing it. :heart: 

Note: over in #403 we just tinkered with the API to provide a generic `Periodogram` parent class and specific `LombScarglePeriodogam` and `BoxLeastSquaresPeriodogram` sub classes.  The latter hold extra data and methods which are specific to those methods.  I imagine psd normalization can go in the parent class (@christinahedges ?), but either way you will want to make sure to start a PR from the most recent version of the `master` branch because of these changes!

**Comment by keatonb** [comment]
This can be closed with #416 having been merged
